app/pages/profile/comment-link.cjsx

```diff
@@ -59,11 +59,14 @@ module.exports = React.createClass
             <a href={@state.href}>
               {if @state.project? and @state.owner
                 <span>
-                  <strong className="comment-project" title="#{@state.owner.display_name}/#{@state.project.display_name}">{@state.project.display_name}</strong>
-                  ➞
+                  {if !@state.project?
```

Comment:
yes that doesn't look right does it, something must have got messed up in my merge from the other branch. will review.